charts/telegraf/values.yaml

```diff
@@ -109,6 +109,17 @@ serviceAccount:
   name:
   # Annotations for the ServiceAccount
   annotations: {}
+
+serviceMonitor:
```

Comment:
Should be mentioned here what specific purpose is this intended for. E.g. Use specific/default outputs.prometheus_client config to match this sample settings etc.

Reply:
Done